functions/functionsv2/ocr/app/process.go

```diff
@@ -32,23 +34,24 @@ func init() {
 // ProcessImage is executed when a file is uploaded to the Cloud Storage bucket you
 // created for uploading images. It runs detectText, which processes the image for text.
 func ProcessImage(ctx context.Context, cloudevent event.Event) error {
-	var event GCSEvent
 	if err := setup(ctx); err != nil {
 		return fmt.Errorf("ProcessImage: %v", err)
 	}
-	if err := cloudevent.DataAs(&event); err != nil {
-		return fmt.Errorf("Failed to parse CloudEvent data: %v", err)
+
+	data := &storagedata.StorageObjectData{}
```

Comment:
See above previous, re: declaring with `var` instead of initializing to an empty object pointer.